redaxo/src/addons/phpmailer/install.php

```diff
@@ -27,4 +27,5 @@
     $this->setConfig('username', '');
     $this->setConfig('password', '');
     $this->setConfig('smtp_debug', '0');
+    $this->setConfig('backup', 1);
```

Comment:
Am besten noch in die update.php rein, aktuell würden bei allen nach dem Update die Backups ausgeschaltet werden, wenn ich das richtig sehe.

Reply:
ist gemacht .. bin ich durch ? :)